server/src/pages/shot/editor.js

```diff
@@ -142,7 +142,7 @@ exports.Editor = class Editor extends React.Component {
       </div>
       <div className="shot-alt-actions annotation-alt-actions">
         <Localized id="annotationSaveButton">
-          <button className="button primary save" id="save" onClick={ this.onClickSave.bind(this) }>Save</button>
+          <button className="button primary save" id="save" onClick={ this.onClickSave.bind(this) } ref={(saveButton) => { this.saveButton = saveButton }}>Save</button>
```

Comment:
You can do this with using `ref`.  (See https://reactjs.org/docs/refs-and-the-dom.html for more `ref`.)

`disabled={this.state.ofWhatever}` and then set the appropriate state in `onClickSave()`.